network-upgrades/ecosystem-readiness.md

```diff
@@ -1,64 +1,72 @@
-## Ecosystem Readiness Checklist
+# Ecosystem Readiness Checklist
 Tooling, Libraries and other Infrastructure
 
-If you know about a status update please add a PR to this document.
+If you know about a status update please add a PR to this document or post on the latest [update issue](https://github.com/ethereum/eth1.0-specs/issues/198) for aggregated inclusion on a weekly basis.
 
-Legend for status updates:
+## London Hardfork
 
-- `-`: EIP not relevant / work not started
-- 🛠️ : In progress
-- ✅ : Ready
+Tracking: `active`
 
-### Tools
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change for ETH 1.0 chain
```

Comment:
Instead of listing the EIPs here, can we just link the spec? This way if it changes this page doesn't become out of date.

Reply:
Ok, updated.